transactions/inscriptionMint.ts

```diff
@@ -244,6 +253,7 @@ export async function inscriptionMintExecute(executeProps: ExecuteProps): Promis
     availableUtxos: addressUtxos,
     feeRate,
     network,
+    useUnconfirmed: false,
```

Comment:
Note this change. We are no longer allowing the use of unconfirmed UTXOs as inputs for inscription txns.